### PR TITLE
dotCMS/core#23683 Applicaiton-wide labels should not have colors

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* Add asterisk on required fields
* change label text color

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: dotCMS/src/main/webapp/html/portlet/ext/useradmin/view_users.jsp [hunk 1]
```diff
@@ -199,27 +199,47 @@
...
                                           <td><input id="emailAddress" type="text" onkeyup="userEmailChanged()" required="true" invalidMessage="Required." dojoType="dijit.form.ValidationTextBox" /></td>
                                       </tr>
                                       <tr>
                                         <th></th>
                                         <td><button dojoType="dijit.form.Button" onclick="generateSecurePasswordModal()" type="button"><%= LanguageUtil.get(pageContext, "generate.secure.password") %></button></td>
                                       </tr>
                                       <tr>
-                                          <th><%= LanguageUtil.get(pageContext, "Password") %>:</th>
+                                          	<th>
+												<span class="required">
```
Comment (fmontes) [inline dotCMS/src/main/webapp/html/portlet/ext/useradmin/view_users.jsp:231]:
```suggestion
												<label class="required">
```

Plase make sure you use Semantic HTML.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/webapp/WEB-INF/jsp/tag_manager/tag_manager.jsp [hunk 2]
```diff
@@ -548,15 +548,15 @@ td {font-size: 100%;}
         <div class="form-horizontal">
             <dl>
                 <dt>
-                    <label for=""><%= LanguageUtil.get(pageContext, "tag") %>:</label>
+                    <label class="required" for=""><%= LanguageUtil.get(pageContext, "tag") %></label>
```
Comment (alfredo-dotcms) [inline dotCMS/src/main/webapp/WEB-INF/jsp/tag_manager/tag_manager.jsp:551]: you need to specify the `for` attr to **all labels**

**Comment by alfredo-dotcms** [comment]
@zulqarnainvd I noticed you're removing a lot of `:` from labels, is that ok? or it's a mistake done by doing **find and replace** IDE method

image

**Comment by zulqarnainvd** [comment]
> @zulqarnainvd I noticed you're removing a lot of `:` from labels, is that ok? or it's a mistake done by doing **find and replace** IDE method
> 
> image

I removed because we need to match field style according to Angular fields.

**Comment by fmontes** [comment]
We need to QA this a lot!

File: dotCMS/src/main/webapp/WEB-INF/jsp/tag_manager/tag_manager.jsp (hunk 2 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/WEB-INF/jsp/tag_manager/tag_manager.jsp:551]: Noted

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss [hunk 3]
```diff
@@ -33,13 +33,16 @@
...
-        content: $fa-var-asterisk;
-        color: $error;
         font-size: 9px;
         position: relative;
         top: -4px;
         left: 2px;
     }
+
+    &::after {
+        content: $fa-var-asterisk;
```
Comment (fmontes) [inline core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss:43]: image

This is for the font-awesome icons, can be used here, in this case is just `*`

Comment (fmontes) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss:43]: Also this needs some padding-left, 2px

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss [hunk 4]
```diff
@@ -92,8 +95,10 @@
 }
 
 .required2 {
-    color: $red;
-    font-weight: bold;
+    &::after {
+        color: $red;
+        content: $fa-var-asterisk;
+    }
```
Comment (fmontes) [inline core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss:101]: Can we unify this? so we don't have `required` and `required2` classes.

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss [hunk 5]
```diff
@@ -333,7 +338,7 @@ dd {
 
     .dijitButton,
     .fakeDojoButton {
-        margin: 0 calc($basic-padding / 2);
+        margin: 0 $basic-padding / 2;
```
Comment (fmontes) [inline core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss:341]: Check your console on build you should be seeing something like this:

image

I updated those when I bring the dojo SCSS to core-web.

File: dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp [hunk 6]
```diff
@@ -106,7 +106,9 @@ final String siteSearch = APILocator.getIndiciesAPI().loadIndicies().getSiteSear
 
 		<dl class="showScheduler">
 			<dt>
-				<span class="required"></span> <strong><%= LanguageUtil.get(pageContext, "name") %></strong>:
+				<label for="" class="required">
+					<strong><%= LanguageUtil.get(pageContext, "name") %></strong>
```
Comment (fmontes) [inline dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp:110]: We don't need strong here.

File: dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp [hunk 7]
```diff
@@ -116,7 +118,9 @@ final String siteSearch = APILocator.getIndiciesAPI().loadIndicies().getSiteSear
 
 		<dl>
 			<dt>
-				<span class="required"></span> <strong><%= LanguageUtil.get(pageContext, "select-hosts-to-index") %>:</strong> <a href="javascript: ;" id="hostsHintHook">?</a> <span dojoType="dijit.Tooltip" connectId="hostsHintHook" id="hostsHint" class="fieldHint"><%=LanguageUtil.get(pageContext, "hosts-hint") %></span>
+				<label class="required">
+					<strong><%= LanguageUtil.get(pageContext, "select-hosts-to-index") %></strong> <a href="javascript: ;" id="hostsHintHook">?</a> <span dojoType="dijit.Tooltip" connectId="hostsHintHook" id="hostsHint" class="fieldHint"><%=LanguageUtil.get(pageContext, "hosts-hint") %></span>
```
Comment (fmontes) [inline dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp:122]: We don't need strong here.

Comment (fmontes) [reply dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp:122]: And nowhere else.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss (hunk 4 above)
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss:101]: there is some css difference in both classes  that's why I don't change it.

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss (hunk 5 above)
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss:341]: Noted

File: dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp (hunk 6 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp:110]: Noted

File: dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp (hunk 7 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp:122]: Noted

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss (hunk 4 above)
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss:101]: ok sure

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss (hunk 3 above)
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss]: updated

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss (hunk 4 above)
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss]: updated

File: dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp (hunk 6 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp]: updated

File: dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp (hunk 7 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_job_schedule.jsp]: updated

File: core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss (hunk 5 above)
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss]: updated

**Review by fmontes — APPROVED** [review]

**Pull request merged**